gravitee-apim-console-webui/src/management/api/general/user-group-access/members/api-general-members.component.ts

```diff
@@ -71,8 +69,8 @@ export class ApiGeneralMembersComponent implements OnInit {
 
   constructor(
     @Inject(UIRouterStateParams) private readonly ajsStateParams,
-    private readonly apiService: ApiService,
-    private readonly apiMembersService: ApiMemberService,
+    private readonly apiService: ApiV2Service,
+    private readonly apiMembersService: ApiMemberV2Service,
```

Comment:
i would call it `apiMemberService` (singular form, as the naming of the services)